index.php

```diff
@@ -189,6 +189,7 @@ function f( a ) {
 				<section id="custom_html-7" class="widget_text widget widget_custom_html">
 					<div class="textwidget custom-html-widget"><a href="#">community sponsors</a></div>
 				</section>
+				<?php require_once( __DIR__ . '/template-parts/home-info.html' ); ?>
```

Comment:
I think we'll want to require this from `front-page.php`, since `index.php` is intended to be used by all the templates. I'll push a commit to get that working the way I imagined, so that you can view the different templates with URLs like

* `{no page var}` -> `template-parts/front-page.php`
* `?page=front-page` -> `template-parts/front-page.php`
* `?page=attendees` -> `template-parts/attendees.php`
* etc

Let me know what you think when you see it, and we can always make adjustments if you think there's a better way.